koku/sources/test/test_storage.py

```diff
@@ -306,7 +308,7 @@ def test_add_provider_sources_auth_info_with_sub_id(self):
                             auth_header=self.test_header,
                             offset=3,
                             endpoint_id=test_endpoint_id,
-                            source_type='AZURE',
+                            source_type='Azure',
```

Comment:
use `Provider.PROVIDER_AZURE` instead?

Reply:
I wasn't aware that sources and providers were so similar which is why I didn't utilize the provider type constants in the sources file - but it is fixed now 😄